src/sorting/selection_sort.rs

```diff
@@ -1,13 +1,22 @@
 pub fn selection_sort<T: Ord>(arr: &mut [T]) {
     let len = arr.len();
-    for left in 0..len {
+
+    if len < 2 {
+        return;
+    }
```

Comment:
This only makes the code more complex. Note that if `len  < 2`, then we will not enter the loop.